examples/tensorflow_v2_example/client.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+import requests
+
+
+def main():
+    try:
+        value = int(sys.argv[1])
+    except (IndexError, ValueError):
+        print(f'Usage: {sys.argv[0]} [integer]')
+        return
+    payload = {'vector': [value]}
+    r = requests.post('http://localhost:9000/predict', json=payload)
```

Comment:
Maybe use our fabulous HTTPClient here? Or this way is more explicit?

Reply:
Yes, this is more explicit. This shows that our auto-generated service is easy to use from any platform. At the same time I think it will be great to have a separate example for our `HTTPClient` itself. But this should be planned as separate issue.